samples/net/zperf/src/zperf_udp_uploader.c

```diff
@@ -260,7 +260,7 @@ void zperf_udp_upload(const struct shell *shell,
 
 		/* Wait */
 #if defined(CONFIG_ARCH_POSIX)
-		k_busy_wait(K_MSEC(100) * USEC_PER_MSEC);
+		k_busy_wait(USEC_PER_MSEC);
```

Comment:
Why the same initial line gets converted to 2 different lines here and above?

Reply:
I saw a working application with only 1ms for UDP but failed to see a working app with 1ms wait in TCP (thus left the original value).
Note that native_posix is special with time handling, for example calling k_yield() like with any other board will render the app non-working.

Reply:
> I saw a working application with only 1ms for UDP but failed to see a working app with 1ms wait in TCP (thus left the original value).

Well, the commit message doesn't say anything about that. And vice-versa, there're functional changes beyond what's described in the commit message.

I don't usually use native_posix, so I'll approve.